configs/inspection/Idealista.xml

```diff
@@ -1,10 +1,9 @@
-<profile version="1.0">
+<inspections profile_name="Idealista" version="1.0">
   <option name="myName" value="Idealista" />
   <inspection_tool class="AbstractClassNamingConvention" enabled="true" level="ERROR" enabled_by_default="true" />
   <inspection_tool class="AndroidLintContentDescription" enabled="false" level="WARNING" enabled_by_default="false" />
-  <inspection_tool class="AndroidLintWrongConstant" enabled="false" level="ERROR" enabled_by_default="false" />
   <inspection_tool class="AnonymousClassComplexity" enabled="true" level="ERROR" enabled_by_default="true">
-    <option name="m_limit" value="5" />
+    <option name="m_limit" value="3" />
```

Comment:
3 it's an reasonable value for this limit, but the problem is we have a CommandUi that has 3 methods and just for the declaration we are going to have complexity equals to 3.
It's a good moment to decide it, maybe this force us to extract the class into its own file.